lib/intercom/request.rb

```diff
@@ -72,11 +72,11 @@ def execute(target_base_url = nil, token:, read_timeout: 90, open_timeout: 30, a
             parsed_body
           rescue Intercom::RateLimitExceeded => e
             if @handle_rate_limit
-              seconds_to_retry = (@rate_limit_details[:reset_at] - Time.now.utc).ceil
+              seconds_to_retry = ((@rate_limit_details[:reset_at] || Time.now.utc) - Time.now.utc).ceil
```

Comment:
If I understand correctly, this line is basically implementing a default `seconds_to_retry` value in the case that `@rate_limit_details[:reset_at]` is `nil`. Would it make more sense for this default value to be >0? If we are hitting a rate limit, it seems to me like the default behaviour should be to wait for a bit before a retry.

Reply:
if there is a problem with the intercom server, or something like that, where this is not getting reset_at, we don't want to keep sleeping the application because we don't know if we are hitting the rate limit. If we hit the rate limit, the error will be raised where we will try to redo the call. If we hit it multiple times it will just "crash" and we will get a response error in the end user. But yeah...maybe I should convert `Time.now.utc` into 0. Not needed to use that "expensive" solution.